libs/components/modals/src/lib/modules/modal/modal-footer.component.html

```diff
@@ -1,21 +1,23 @@
 <div class="sky-modal-footer-container sky-padding-even-large">
   <div aria-live="polite">
     @if (errorsSvc.formErrors | async; as formErrors) {
-      <div class="sky-modal-footer-errors sky-margin-stacked-lg">
-        @for (formError of formErrors; track formError) {
-          <sky-status-indicator
-            descriptionType="error"
-            indicatorType="danger"
-            [ngClass]="{
-              'sky-margin-stacked-lg':
-                formErrors.indexOf(formError) < formErrors.length - 1,
-              'footer-error': true
-            }"
-          >
-            {{ formError.message }}
-          </sky-status-indicator>
-        }
-      </div>
+      @if (formErrors.length > 0) {
```

Comment:
This is in a follow-up if-statement because of the use of `as formErrors`. We can not use that and also do the length check, but because the first if-statement is `async` I didn't want to get rid of the `as formErrors`.